bootloader/assets/generateasset/main.go

```diff
@@ -0,0 +1,157 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
nitpick: but this I think should either be called something short like "genasset" , otherwise it should be "generate-asset" with a hyphen

Reply:
I'll go with genasset sounds fine and is considerably shorter than generate-asset.